kratos/utilities/read_materials_utility.cpp

```diff
@@ -331,6 +335,47 @@ void ReadMaterialsUtility::AssignTablesToProperty(
 /***********************************************************************************/
 /***********************************************************************************/
 
+void ReadMaterialsUtility::AssignAccessorsToProperty(
+    const Parameters MaterialData,
+    Properties& rProperty
+    )
+{
+    KRATOS_TRY;
+
+    if (MaterialData.Has("Accessors")) {
+        Parameters accessors = MaterialData["Accessors"];
+
+        for (auto iter = accessors.begin(); iter != accessors.end(); ++iter) {
+            auto accessor_param = accessors.GetValue(iter.name());
+
+            // Independent Variable
+            std::string input_var_name = accessor_param["table_input_variable"].GetString();
+            TrimComponentName(input_var_name);
```

Comment:
Since this is new, lets enforce the use of raw var names, i.e. remove this fct call